src/parser/jobs/mnk/Fists.js

```diff
@@ -28,28 +54,51 @@ export default class Fists extends Module {
 
 	static title = 'Fist Stances'
 
-	_stanceApplied = []
-	_stanceDropped = []
-	_stanceless = 0
+	_activeFist = STANCES.find(fist => this.combatants.selected.hasStatus(fist)) || STANCELESS
+	_fistUptime = {[STANCELESS]: 0} // Initialise stanceless to prevent weird UI shit
+	_fistGCDs = {[STANCELESS]: 0} // Initialise empty
+
+	_lastFistChange = this.parser.fight.start_time
 
 	constructor(...args) {
 		super(...args)
+		this.addHook('cats', {by: 'player'}, this._onCast)
```

Comment:
You should probably fix this.

WE DON'T HAVE CATS IN THIS PROJECT.

Reply:
:blobcatmorning:

Reply:
Can we get cats though?